package.json

```diff
@@ -86,7 +87,8 @@
         "react-load-script": "^0.0.6",
         "react-redux": "^5.0.6",
         "react-router-dom": "^5.1.2",
-        "redux": "^4.0.0"
+        "redux": "^4.0.0",
+        "seamless-immutable": "^7.1.4"
```

Comment:
> @kelvin-muchiri Is this being installed because it's a dependency for `@onaio/gatekeeper`?
> 
> It looks like `seamless-immutable` is [a dependency for the `@onaio/session-reducer`](https://github.com/onaio/js-tools/blob/master/packages/session-reducer/package.json#L36) package, which is [a dependency of `@onaio/gatekeeper`](https://github.com/onaio/js-tools/blob/master/packages/gatekeeper/package.json#L34).
> 
> It feels like we shouldn't have to add this here and that it should be handled simply by installing gatekeeper.

Sure. Its a dependency for `@onaio/gatekeeper`

Reply:
Ok cool, let's keep it in our `package.json` for now and keep an eye out for an update from Gatekeeper to see if we can remove it.

Reply:
@cKellyDesign @kelvin-muchiri seems like it can be inferred there is some gatekeeper work that needs to be done.  Is that so?  If yes, could we have an issue please?